trieste/acquisition/function.py

```diff
@@ -888,9 +889,35 @@ class ExpectedHypervolumeImprovement(SingleModelAcquisitionBuilder):
     follows :cite:`yang2019efficient`
     """
 
+    def __init__(
+        self,
+        ref_point_specification: Sequence[float]
```

Comment:
The parameter name reads a bit weird to me. How about this?

```suggestion
        reference_point_spec: Sequence[float]
```